rotkehlchen/rotkehlchen.py

```diff
@@ -435,6 +441,15 @@ def deactivate_premium_status(self) -> None:
         self.accountant.deactivate_premium_status()
         self.chain_manager.deactivate_premium_status()
 
+    def activate_premium_status(self, premium: Premium) -> None:
+        """Activate the premium in the current session if was deactivated"""
```

Comment:
I specifically said here. Please do not ignore PR comments.
```suggestion
        """Activate premium in the current session if was deactivated"""
```